frontend/src/app/projects/service/project.service.ts

```diff
@@ -142,3 +153,8 @@ export const ProjectVisibility = {
   private:
     'Private - users need to be explicitly granted access to see the project.',
 };
+
+export const ProjectTypes = {
```

Comment:
I was first really confused (also because of the `ProjectVisibility`) but wouldn't it be better to name this here `ProjectTypeDescriptions`? Because right now, we would have `ProjectType` and `ProjectTypes` and when only seeing the names I would not know what they are used for and whats the difference between them.
(It is even more confusing in case of the `ProjectVisibility` as there the type and const have the exact same name but thats not part of this PR)

Probably I am missing something here, so feel free to drop your reasoning/opinion here.